raiden/tests/utils/network.py

```diff
@@ -313,7 +313,7 @@ def create_apps(
                                 'server': 'matrix.local.raiden',
                             },
                             'retries_before_backoff': retries_before_backoff,
-                            'retry_interval': retry_interval,
+                            'retry_interval': retry_interval * 5,
```

Comment:
this is not the right place for this change, please change it on `raiden.tests.fixtures.variables`

Reply:
Problem is that that fixture doesn't have yet access to the transport. Will look on how to inject that there